webapp/src/modules/estates/sagas.js

```diff
@@ -185,14 +185,15 @@ function* handleTransferRequest({ estate, to }) {
 
     const contract = eth.getContract('EstateRegistry')
     const txHash = yield call(() =>
-      contract.safeTransferFrom(oldOwner, to, estate.token_id)
+      contract.safeTransferFrom(oldOwner, to, estate.id)
     )
 
     const transfer = {
       txHash,
       oldOwner,
       to,
-      estate
+      estate: { id: estate.id, data: { name: estate.data.name } },
```

Comment:
same as estates/actions.js

Reply:
Sorry, I don't get it. What do you mean? We are not passing `parcels` here.

Reply:
Added comments as we talked offline